src/status_im/translations/ne.cljs

```diff
@@ -0,0 +1,314 @@
+
+   ;;drawer
+   :invite-friends                        "साथीहरुलाई निम्त्याउं"
+   :faq                                   "प्राय सोधिने प्रश्नोत्तर"
+   :switch-users                          "उपयोगकर्ता बदलौं"
+   :feedback                              "प्रतिक्रिया"
+   :view-all                              "सबै हेर्ने"
+   :current-network                       "चल्ती नेटवर्क"
+
+   ;;chat
+   :is-typing                             "ले टाइप गर्दै हुनुहुन्छ"
+   :and-you                               "अनि तपाईं"
+   :search-chat                           "च्याटमा खोजौं"
+   :members                               {:one "१ सदस्य"}
+   :other                                 "{{count}} सदस्यहरु"
+   :zero                                  "सदस्यहीन"
+   :members-active                        {:one   "१ सदस्य"
+                                           :other "{{count}} सदस्यहरु"}
+   :zero                                  "सदस्यहीन"
+   :public-group-status                   "सार्वजनिक"
+   :active-online                         "अनलाइन"
+   :active-unknown                        "अज्ञात"
+   :available                             "उपलब्ध"
+   :no-messages                           "सन्देशहरु खाली"
+   :suggestions-requests                  "अनुरोधहरु"
+   :suggestions-commands                  "आदेशहरु"
+   :faucet-success                        "धारा अनुरोध प्राप्त भएको छ"
+   :faucet-error                          "धारा अनुरोध त्रुटि"
+
+   ;;sync
+   :sync-in-progress                      "सिङ्क भइरहेको छ॰॰॰"
+   :sync-synced                           "सिङ्क मा छ"
+
+   ;;messages
+   :status-sending                        "पठाउंदै"
+   :status-pending                        "बाँकी छ"
+   :status-sent                           "पठाइ सकेको"
+   :status-seen-by-everyone               "सबैले हेरिसकेको"
+   :status-seen                           "देखि सकेको"
+   :status-delivered                      "बुझाइएको"
+   :status-failed                         "असफल"
+
+   ;;datetime
+   :datetime-ago-format                   "{{number}} {{time-intervals}} {{ago}}"
+   :datetime-second                       {:one   "सेकेन्ड"
+                                           :other "सेकेन्ड"}
+   :datetime-minute                       {:one   "मिनेट"
+                                           :other "मिनेट"}
+   :datetime-hour                         {:one   "घण्टा"
+                                           :other "घण्टा"}
+   :datetime-day                          {:one   "दिन"
+                                           :other "दिन"}
+   :datetime-multiple                     "से"
+   :datetime-ago                          "पहिले"
+   :datetime-yesterday                    "हिजो"
+   :datetime-today                        "आज"
+
+   ;;profile
+   :profile                               "प्रोफाइल"
+   :edit-profile                          "प्रोफाइल सम्पादन"
+   :report-user                           "उपयोगकर्तालाई रिपोर्ट गर्नुहोस्"
```

Comment:
उपयोगकर्ताबारे सिकायत पठाउनुहोस् love it 😄 !

wonder if there are even better suited synonyms for सिकायत 🤔